packages/asm-lsp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/bergercookie/asm-lsp
+TERMUX_PKG_DESCRIPTION="language server for NASM/GAS/GO assembly"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.6.0"
```

Comment:
Seems like you're missing
```suggestion
TERMUX_PKG_VERSION="0.6.0"
TERMUX_PKG_DEPENDS="openssl"
```

Reply:
Fixed.